src/app/server/CommissioningWindowManager.h

```diff
@@ -30,8 +30,16 @@
 #include <protocols/secure_channel/PASESession.h>
 #include <system/SystemClock.h>
 
+#include <app/server/PlatformDnssdServer.h>
```

Comment:
If this is what gets injected elsewhere, it's not needed as a dependency here. Please remove this dependency and only depend on DnssdServer.